script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_4.txt

```diff
@@ -44,7 +44,8 @@ Why did I try to go this far back...?
 <Page10>
 "Sorry. Do you have a hundred yen on you?"
 When I turn around, I see a strange girl in front of the vending machine.
-'I've been trying to buy some juice... but I'm stuck without any change.' she says.
+As if to say, 'I've been trying to buy some juice... but I'm stuck without any change.'//ジュースを買おうとしたが小銭がなく立ち往生している……といった<体|てい>だ。
```

Comment:
I think the といった体だ got a little too dropped here? It's not clear what is doing the conveying of meaning here (her posture/appearance)

Reply:
Mmm, I think it's okay? My change was to remove the "she says" because she's not saying anything with words. 
But we could do something like
>She is standing there in a way that screams 'I've been trying to buy some juice... but I'm stuck without any change.'

?